example-site/components/TemplateBanner.tsx

```diff
@@ -15,6 +15,7 @@ export const TemplateBanner = ({ name, slug }: TemplateBannerProps) => {
 			background="body"
 			flexDirection={['column', 'row']}
 			gap={[2, 1]}
+			id="template-banner"
```

Comment:
I had to change the way image generation was done and needed a styling hook for this